common/api-review/auth-exp.api.md

```diff
@@ -39,17 +40,17 @@ export function applyActionCode(auth: externs.Auth, oobCode: string): Promise<vo
 // @public (undocumented)
 export class AuthCredential {
     protected constructor(providerId: string, signInMethod: string);
-    // Warning: (ae-forgotten-export) The symbol "AuthCore" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-forgotten-export) The symbol "Auth" needs to be exported by the entry point index.d.ts
     // Warning: (ae-forgotten-export) The symbol "PhoneOrOauthTokenResponse" needs to be exported by the entry point index.d.ts
     //
     // (undocumented)
-    _getIdTokenResponse(_auth: AuthCore): Promise<PhoneOrOauthTokenResponse>;
+    _getIdTokenResponse(_auth: Auth_2): Promise<PhoneOrOauthTokenResponse>;

```

Comment:
Auth_2?

Reply:
Yeah that's weird, but I think it references model.Auth instsead of externs.Auth. It should only be visible to internal methods so it should go away once we signal the documentation to ignore symbols prefixed with `_`.